rotkehlchen/db/reports.py

```diff
@@ -226,14 +227,21 @@ def add_report_data(
         - InputError if the event can not be written to the DB. Probably report id does not exist.
         """
         data = event.serialize_for_db(ts_converter)
+
+        pnl_taxable = str(event.pnl.taxable)
+        try:
+            asset = event.asset.symbol_or_name()
+        except WrongAssetType:
+            asset = 'UNKNOWN'
```

Comment:
why not just store it as empty string?

Reply:
Also why not make the schema allow nullable and have them null if we have assets without symbol?

Reply:
Makes sense store as null, I will change the schema to accept it